### ISSUE TITLE
verify(0, $matcher) never throws an error

### ISSUE THREAD

**Issue by TerjeBr** [rowanhill/wiremock-php#1]
I have this code:
```
$matcher = WireMock::postRequestedFor(
            WireMock::urlEqualTo("/some/url")
        );
$this->wiremock->verify(0, $matcher);
```

This does not throw any error when I am 100% sure wiremock has received something at "/some/url" and thus the count should not be 0.

**Comment by TerjeBr** [comment]
@rowanhill ping

**Comment by rowanhill** [comment]
Interesting. I haven't looked at this project in quite a long time, but I would have expected that to work.

I'll try and find some free time to take a look. It'd be great if you could put together an entire failing test (along the lines of https://github.com/rowanhill/wiremock-php/blob/master/test/WireMock/Integration/VerificationIntegrationTest.php#L100) and we can see if I can reproduce it?

You could also triple-check that you're definitely hitting WireMock as you intend by making a request (with curl, or postman or similar) to the server directly to query for the number of matching requests it's seen, as per http://wiremock.org/verifying.html#via-json-http.

Something like:
```bash
curl -XPOST -d '{
    "method": "POST",
    "url": "/some/url"
}' 'http:localhost:8080__admin/requests/count'
```

**Comment by rowanhill** [comment]
Okay, I've had a chance to test it now - it does look like a bug, and I have a fix.

I'll try and release a fix ASAP.

**Comment by rowanhill** [comment]
This should be fixed in commit 7b54820. That's available in packagist as version [1.43.1](https://packagist.org/packages/wiremock-php/wiremock-php#1.43.1).

Please let me know if you're still having problems.

**Comment by TerjeBr** [comment]
Thank you. I appreciate it.
